tests/testthat/test-correlation.R

```diff
@@ -237,6 +242,16 @@ test_that("correlation returns an appropriately structured result for assay agai
   #expect_equal(as.character(unique(statsData$data1)), names(testOTU)[2:(length(names(testOTU)) - 1)])
   #expect_equal(as.character(unique(statsData$data2)), names(testOTU)[3:length(names(testOTU))])
   expect_true(all(!is.na(statsData)))
+
+  # method = sparcc
+  result <- selfCorrelation(data, method='sparcc', verbose = FALSE)
```

Comment:
this should fail with the negative data passed to it, no?

Reply:
what negative data? df is our testOTU here i think.